### PR TITLE
define MAP_WHEEL

### PR DESCRIPTION
**Pull request by hidemune**
I avoided defining a new type.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Only for Mad Cats Steering Wheel xbox one
```
Comment (cgutman) [inline README.md]: Please undo changes to this file.

File: replace.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (cgutman) [inline replace.sh]: Please remove this file from your PR.

File: xpad.c [hunk 3]
```diff
@@ -75,6 +75,7 @@
  * Later changes can be tracked in SCM.
  */
 #define DEBUG
+#define DEBUG_VERBOSE
```
Comment (cgutman) [inline xpad.c]: Please remove this.

File: xpad.c [hunk 4]
```diff
@@ -231,6 +233,8 @@ static const struct xpad_device {
 	{ 0x24c6, 0x5b03, "Thrustmaster Ferrari 458 Racing Wheel", 0, XTYPE_XBOX360 },
 	{ 0x1532, 0x0a03, "Razer Wildcat for Xbox One", 0, XTYPE_XBOXONE },
 	{ 0xffff, 0xffff, "Chinese-made Xbox Controller", 0, XTYPE_XBOX },
+	{ 0x0738, 0x4503, "Mad Catz Pro Racing Force Feedback Wheel & Pedals (Xbox One)", MAP_WHEEL, XTYPE_XBOXONE },
```
Comment (cgutman) [inline xpad.c]: Please put this in order in the device list (and avoid the extra blank line)

File: xpad.c [hunk 5]
```diff
@@ -1379,7 +1388,9 @@ static void xpad_set_up_abs(struct input_dev *input_dev, signed short abs)
 		break;
 	case ABS_Z:
 	case ABS_RZ:	/* the triggers (if mapped to axes) */
-		if (xpad->xtype == XTYPE_XBOXONE)
+		if (xpad->mapping & MAP_WHEEL)
```
Comment (cgutman) [inline xpad.c]: Can you please put a comment with details about which physical devices are mapped to which `ABS_` values and ranges and center of those values?

Comment (cgutman) [inline xpad.c]: I think my preference would be:

wheel - ABS_X (-32768, center at 0, 32768)
brake - ABS_Z (0 to 65535)
gas - ABS_RZ (0 to 65535)

**Comment by hidemune** [comment]
Thank you for your advice.

**Pull request closed without merging**